README.md

```diff
@@ -14,243 +14,222 @@
+
+- Name: The name of your zone, e.g. garden
+- Size: The size of this zone (m<sup>2</sup> or sq ft)
+- Throughput: The flow of this zone (liter/minute or ???/minute)
+- Bucket: You can manipulate this calculated value.
+- Lead Time: In seconds. Time needed to warm up your irrigation system, e.g. time to establish a connection or start a pump etc.
+
+You can Update all zones and Callculate all.
+
+**Per Zone Settings**
+You can change any value mentioned before. Additionaly there are some more options.
+
+- State: Wether Updating and Calculation of that zone should be enabled in mode 'Automatic' or 'Manual' or 'Disabled'
+- Module: Choose the calculation module (see below) shall be used for that zone to calculate irrigation duration
+- Mapping: Which sensor mapping (see below) shall be used for that zone
+- Bucket: Either calculated or manualy set. Bucket >=0 no irrigation is necesarry, bucket <0 irrigation is necesarry.
+- Multiplier: Multiplies the duration of irrigation
+- Duration: Either calculated or manualy set
+
+Below each zone there are some buttons to update, calculate or delete that zone.
+
+#### MODULES
+Add one or more modules that calculate irrigation duration. Each module comes with its own configuration and can be used to calculate duration for one or more zones. Modules can't be deleted if they are used by one or more zones.
+
+- 0:PyETO: Calculate duration based on the FAO56 calculation from the PyETO library. 
+  - If you set PyETO to not estimate, it will look for a solar radiation sensor in the mapping and will use that value. If there is none, it will use OWMs value (assuming you have OWM configured).
+If you let PyETO to estimate from temperature or sun hours, it will not ask OWM for a solar radiation value nor will it ask a sensor for a solar radiation value, even if you configured it in the mapping.
+  - Coastal: If the location you are tracking is situated on or adjacent to coast of a large land mass or anywhere else where air masses are influenced by a nearby water body, enable this setting.
+  - Solrad behaviour: Should solar radiation estimated from temperature or sun hours or disabled 
+  - Forecast days: How many forecast days taken into account
+- 1:Static: 'Dummy' module with a static configurable duration.
+  - Delta: ???
+
+#### MAPPINGS
+For sensor configuration take care to make sure the unit the component expects is the same as your sensor provides.**
+
 - Names of sensors that supply required measurements (optional). Only required in mode 2) and 3). See [Measurements and Units](https://github.com/jeroenterheerdt/HAsmartirrigation/wiki/Measurements-and-Units) for more information on the measurements and units expected by this component.
```

Comment:
I think we should not talk about modes any more. Also, wiki needs updating as the units are now configurable on the mapping, as well as the aggregation.

Reply:
Removed it. Will have a look at wiki.

Reply:
We have to find a way editing wiki